ebau_gwr/core/serializers.py

```diff
@@ -1,9 +1,93 @@
 from rest_framework import serializers
 
 
+class DateField(serializers.DateField):
```

Comment:
What's the difference to the parent class? Why is this needed?

Reply:
Hmm. Might be useful to name it `PyxbDateField` then, to avoid confusion

Reply:
Done.